azure-mgmt-monitor/src/main/java/com/microsoft/azure/management/monitor/LocalizableString.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.monitor;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+import com.microsoft.azure.management.apigeneration.LangMethodDefinition;
+
+/**
+ * The localizable string class.
+ */
+@LangDefinition(ContainerName = "/Microsoft.Azure.Management.Monitor.Fluent.Models")
+public interface LocalizableString {
```

Comment:
`extends HasInner<LocalizableStringInner>` so that inner() exposed by LocalizableStringImpl (via WrapperImpl) is visible to user